### ISSUE TITLE
Fix encoding issue with Etherpad

### ISSUE THREAD

**Issue by cdparra** [socialappslab/appcivist#7]

**Comment by cdparra** [comment]
@LucileMP, after much trying to fix this, we have concluded this is an issue with Etherpad. We have opened [an issue on Etherpad's repository](https://github.com/ether/etherpad-lite/issues/3184#issuecomment-299519265). We are waiting for ETH developers to fix it. 

In parallel, we have also started to explore alternatives to etherpad and how to integrate them (including GDocs). This will be followed up on [this appcivist-pb-client issue](https://github.com/socialappslab/appcivist-pb-client/issues/537)

**Comment by cdparra** [comment]
@LucileMP: since we are getting no answer from etherpad, and the subcontract with AppCivist continues to be unfulfilled, we are putting this on hold.

**Comment by cdparra** [comment]
[Possible solution suggestion](https://github.com/ether/etherpad-lite/issues/3184#issuecomment-312640442), to try out within appcivist.

**Comment by cdparra** [comment]
@LucileMP we tried out a solution that works. The update will be available over the weekend.
